CMakeLists.txt

```diff
@@ -125,6 +125,12 @@ set(FDB_FUTURE_VERSION "7.5.0")
 set(FDB_PREV_RELEASE_VERSION "7.3.41")
 set(FDB_PREV2_RELEASE_VERSION "7.1.33")
 
+set(NO_MULTIREGION_TEST OFF CACHE BOOL "When set, disable simulation tests using multiple regions.")
```

Comment:
Using an `option` might be a better choice, see:

https://stackoverflow.com/questions/36358217/what-is-the-difference-between-option-and-set-cache-bool-for-a-cmake-variabl